app/src/main/java/org/mifos/mobile/ui/activities/splash.java

```diff
@@ -1,27 +1,38 @@
 package org.mifos.mobile.ui.activities;
 
-/*
- * Created by saksham on 01/June/2018
- */
+import androidx.appcompat.app.AppCompatActivity;
 
 import android.content.Intent;
 import android.os.Bundle;
+import android.os.Handler;
 
 import com.mifos.mobile.passcode.utils.PasscodePreferencesHelper;
 
-import org.mifos.mobile.ui.activities.base.BaseActivity;
+import org.mifos.mobile.R;
 import org.mifos.mobile.utils.Constants;
 
-public class SplashActivity extends BaseActivity {
+public class splash extends AppCompatActivity {
```

Comment:
Please change the name so that it starts with capital letter 
I don't see a point why you changed the parent class . I would suggest you to undo this